docs/content/topics/administrators/authentication.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: Authentication
 description: Authentication configuration for Brigade
+aliases:
+  - /authentication
+  - /topics/authentication.md
+  - /topics/administrators/authentication.md
```

Comment:
Isn't this an operator concern?

Reply:
This is one that definitely straddles both roles.  I would think admins would want a choice over which third-party auth system is deployed (by an operator), as it relates to users which they'll set up authz around... but I'm down to move it if we feel strongly.

Reply:
Interesting point. I can imagine admins wanting some say in the matter. Let's roll with it as is for now and we can change it later if necessary.